resources/api/openapi.yml

```diff
@@ -0,0 +1,382 @@
+openapi: 3.0.3
+
+info:
+  version: 0.0.1-beta
+  title: Engelsystem
+  description: |
+    This API is as stable as a **beta** version might be.
+    It could burst into flames and morph into a monster at any second!
+    (But we try to keep it somewhat consistent, at least during events).
+  contact:
+    name: GitHub Issues
+    url: https://github.com/engelsystem/engelsystem/issues
+  license:
+    name: GPL 2.0
+    # identifier: GPL-2.0
+
+servers:
+  - url: /api/v0-beta
+    description: This server
+  - url: http://localhost:5080/api/v0-beta
+    description: Your local dev instance
+
+tags:
+  - name: news
+    description: News and meeting announcements
+  - name: shift
+    description: Event shifts and location
+
+components:
+  securitySchemes:
+    bearerAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: API key from settings
+
+  responses:
+    UnauthorizedError: # 401
+      description: Access token is missing or invalid
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Error'
+    ForbiddenError: # 403
+      description: The client is not allowed to access
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Error'
+    NotImplementedError: # 501
+      description: This endpoint or method is not implemented
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Error'
+
+  schemas:
+    AngelType:
+      type: object
+      properties:
+        id:
+          type: integer
+          example: 42
+        name:
+          type: string
+          example: Angel
+        description:
+          type: string
+          example: Meta-Group of all registered Angels
+        url:
+          type: string
+          example: https://example.com/news/42
```

Comment:
This example URL does not look correct. Can you add a description to the URL fields to explain where they are leading to? Such as „Browsable URL of the angel type in the Engelsystem“.